test/browser.js

```diff
@@ -25,8 +25,8 @@ function createRepo (id, done) {
     })
   })
 
-  const mainBlob = new Store(id)
-  const blocksBlob = new Store(`${id}/blocks`)
+  const mainBlob = new Store(id + Math.random())
+  const blocksBlob = new Store(`${id}/blocks` + Math.random())
```

Comment:
This was due to https://github.com/ipfs/js-ipfs/issues/528#issuecomment-255138513